src/libwalletqt/Wallet.h

```diff
@@ -294,8 +333,13 @@ class Wallet : public QObject
     int     m_connectionStatusTtl;
     mutable QTime   m_connectionStatusTime;
     mutable bool    m_initialized;
+    uint32_t m_currentSubaddressAccount;
     AddressBook * m_addressBook;
     mutable AddressBookModel * m_addressBookModel;
+    Subaddress * m_subaddress;
+    mutable SubaddressModel * m_subaddressModel;
+    SubaddressAccount * m_subaddressAccount;
+    mutable SubaddressAccountModel * m_subaddressAccountModel;
```

Comment:
These mutable qualifiers would like go with the const on the getter above removed

Reply:
Removed